src/odemis/gui/comp/miccanvas.py

```diff
@@ -1055,14 +1062,36 @@ def _convert_streams_to_images(self):
         self.merge_ratio = self.view.stream_tree.kwargs.get("merge", 1)
 
         # always refit to image (for the rare case it has changed size)
-        self.fit_view_to_content(recenter=True)
+        self.fit_view_to_content(recenter=False)
 
     def on_size(self, event):
         # refit image
-        self.fit_view_to_content(recenter=True)
+        self.fit_view_to_content(recenter=False)
         # Skip DblMicroscopeCanvas.on_size which plays with mpp
         canvas.DraggableCanvas.on_size(self, event)
 
+    def fit_to_content(self, recenter=False):
+        # Override the default function to _not_ move the center, as we always
+        # display everything at 0,0.
+
+        bbox = self._getContentBoundingBox()
+        if bbox[0] is None:
+            return  # no image => nothing to do
+
+        # compute mpp so that the bbox fits exactly the visible part
+        w, h = bbox[2] - bbox[0], bbox[3] - bbox[1]  # m
+        if w == 0 or h == 0:
+            logging.warning("Weird image size of %fx%f m", w, h)
```

Comment:
"%" missing?  -> % (w, h)?

Reply:
All the logging.* functions accepts as extra parameters the values to format into the string. Actually, it's even the recommended way of writing it.